src/include/piomatter/matrixmap.h

```diff
@@ -134,31 +134,26 @@ schedule_sequence make_temporal_dither_schedule(int n_planes,
     if (n_temporal_planes >= n_planes) {
         throw std::range_error("n_temporal_planes can't exceed n_planes");
     }
-    if (n_temporal_planes != 2 && n_temporal_planes != 4) {
-        // the code can generate a schedule for 8 temporal planes, but it
-        // flickers intolerably
-        throw std::range_error("n_temporal_planes must be 0, 1, 2, or 4");
-    }
 
     int n_real_planes = n_planes - n_temporal_planes;
 
-    schedule base_sched;
-    for (int j = 0; j < n_real_planes; j++) {
-        base_sched.emplace_back(
-            9 - j, (1 << (n_temporal_planes + n_real_planes - j - 1)) /
-                       n_temporal_planes);
-    }
-
     schedule_sequence result;
 
-    auto add_sched = [&result, &base_sched](int plane, int count) {
-        auto sched = base_sched;
+    auto add_sched = [&result, n_real_planes,
+                      n_temporal_planes](int i, int plane, int count) {
+        schedule sched;
+        for (int j = 0; j < n_real_planes; j++) {
+            int k = 1 << (n_temporal_planes + n_real_planes - j - 1);
+            printf("n_real_planes=%d n_temporal_planes=%d plane=%d k=%d\n",
```

Comment:
Is this print statement added and left enabled on purpose? I don't really have an opinion on keeping it or not, but saw some repeated printouts from it and wanted to double check that it was intentional

Reply:
and now it is.